### DISCUSSION TITLE
How can i calculate total amount of dynamically created location amount

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9191] [Q&A] [answered]
When i add location dynamically using usefieldArray after that i want to calculate the total amount based on sum of amount fields
https://codesandbox.io/s/property-form-updated-forked-1f2460?file=/src/components/form/property_location.jsx

**Comment by bluebill1049** [comment, marked as answer]
answered: https://github.com/react-hook-form/react-hook-form/discussions/9196
